packages/flutter/lib/src/widgets/content_commit.dart

```diff
@@ -0,0 +1,54 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
I think the name of this file and the name of the class should match, unless you have a reason for them not to.  So maybe rename this file to "committed_content.dart".